katrain/__main__.py

```diff
@@ -5,6 +5,14 @@
 
 os.environ["KCFG_KIVY_LOG_LEVEL"] = os.environ.get("KCFG_KIVY_LOG_LEVEL", "warning")
 
+import platform
+
+if platform.system() == "Windows":
```

Comment:
I believe kivy has a platform detection module as well which is used at some points, can we use that one instead to keep things consistent?

Reply:
I changed it to use kivy.platform, which was already imported.